src/android/Capture.java

```diff
@@ -301,6 +328,16 @@ private void captureVideo(Request req) {
             PermissionHelper.requestPermission(this, req.requestCode, Manifest.permission.CAMERA);
         } else {
             Intent intent = new Intent(android.provider.MediaStore.ACTION_VIDEO_CAPTURE);
+            String timeStamp = new SimpleDateFormat("yyyyMMddHHmmssSSS").format(new Date());
+            String fileName = "VID_" + timeStamp + ".avi";
```

Comment:
I suggest `".mp4"` instead of `".avi"` as it the file extension I get for video capture with previous code without these FileProvider changes

Reply:
Is it save to assume file extension to begin with? I imagine this could differ depending on the underlying camera app.

Reply:
@breautek I think you're right :/

"but" one more link in Android training docs should be interesting to follow : https://developer.android.com/guide/topics/media/camera.html#saving-media
for its `getOutputMediaFile()`, with output switch for MEDIA_TYPE_IMAGE or MEDIA_TYPE_VIDEO = "IMG_*.jpg" // "VID_*.mp4"

a similar output extension of `.mp4` can be found in CameraX training too

Reply:
Ah, well if the docs shows an assumption of `.mp4` I guess it's fine to assume!